maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -160,4 +166,22 @@ private void report(Bndrun run) {
 		}
 	}
 
+	private void attach(File file) {
+		if (!attach) {
+			logger.debug(
+					"The export plugin has been configured not to attach the generated application to the project.");
+			return;
+		} else if (bundlesOnly) {
+			logger.debug("The export plugin will not attach a bundles-only output to the project.");
+			return;
+		}
+
+		DefaultArtifactHandler handler = new DefaultArtifactHandler("jar");
+		handler.setExtension("jar");
+		DefaultArtifact artifact = new DefaultArtifact(project.getGroupId(), project.getArtifactId(),
```

Comment:
Won't N executable jar files end up having the same `artifactId`? I wonder if the `artifactId` should maybe incorporate the file name to make it unique?

Reply:
We could use the bndrun file name as the classifier to ensure uniqueness. Changing the artifactId is probably a bad plan as it breaks the link between the module GAV and its published output...

Reply:
yes that makes more sense.